spec/factories/charges.rb

```diff
@@ -1,13 +1,17 @@
 require 'faker'
 
 FactoryGirl.define do
-  factory :charge do |f|
-    f.jms_charge_id "12345"
-    f.booking_id "98765"
-    f.code "02015"
-    f.description "theft"
-    f.category "misdemeanor"
-    f.court_case_number "13579-ABC"
-    f.bond_amount 250.00
+  factory :charge do
+    jms_charge_id '12345'
+    booking_id '98765'
+    code '02015'
+    description 'theft'
+    category 'misdemeanor'
+    court_case_number '13579-ABC'
+    bond_amount 250.00
```

Comment:
If there aren't substantive changes in here, it might be easiest to revert them in this branch and only keep the new trait.